tests/dbt/test_transformation.py

```diff
@@ -104,6 +104,7 @@ def test_model_kind():
 
 def test_model_columns():
     model = ModelConfig(
+        alias="test",
```

Comment:
This was added because we're hitting a parse error due to the model's name being an empty string => we're trying to parse an empty string into a `Table` inside the model's `"name"` validator and it raises a parse error.